deployer/README.md

```diff
@@ -462,6 +484,32 @@ To set the parameters:
       oc create -n logging -f -
     # this creates the daemonset and starts fluentd with the new params
 
+#### Have Fluentd use the systemd journal as the log source
+
+By default, fluentd will read from `/var/log/messages*` and
+`/var/log/containers/*.log` for system logs and container logs, respectively.
+You can use the systemd journal instead as the log source.  There are three new
```

Comment:
the word "new" has a way of sticking around long after it's ceased to be new. How about "related"